tests/cpp/test_indexing.cpp

```diff
@@ -35,6 +37,26 @@ std::vector<Val*> getLoopIndices(TensorView* tv, const TensorIndexer& indexer) {
   return loop_indices;
 }
 
+template <typename... Args>
+Val* addExpr(Args&&... args) {
+  return SimplifyingIrBuilder::addExpr(std::forward<Args>(args)...);
```

Comment:
These do help readability. If you're planning to do a lot of structural checking of indices we could consider using [user-defined literals](https://en.cppreference.com/w/cpp/language/user_literal) like what @zasdfgbnm used in `test_expr_simplifier.cpp`.